tests/e2e/base.test.js

```diff
@@ -1,15 +1,16 @@
 /* global fixture */
 import { runBaseTests } from './helpers/base';
 import BookReader from './models/BookReader';
+import { LOCAL_URL } from './constants';
 
-const localURL = 'http://127.0.0.1:8000/BookReaderDemo/';
```

Comment:
Let's not pull this out; this would be better as an environment variable or parameter. Having it in a separate file will make that significantly more difficult.

Reply:
I agree on the env variable (I use an env variable in Petabox's TestCafe tests). Do you want that set in the NPM scripts for testcafe test running, or should we bring in a package like [dotenv](https://www.npmjs.com/package/dotenv) to bring them in from a local file?

Reply:
Your call how much you want to do as part of this PR. I think having a basic environment variable, which defaults to localhost is best in the spirit of yagni. If we end up with a lot of environment variables, then we should consider something more advanced like dotenv.